Add an identifier for acmart's nonacm flag.

LeifAndersen opened the pull request.

bennn commented on scribble-lib/scribble/acmart/lang.rkt:
```diff
@@ -22,12 +22,13 @@
            [authorversion? #f]
-           [font-size #f])
+           [font-size #f]
+           [nonacm? #f])
```
Its not a boolean-valued variable, so change the name to `nonacm` ?

LeifAndersen replied:
I would, but in this case I thought it better to follow the convention of the other variables. So maybe we could remove the `?` on all of them?

bennn replied:
Oh I didn't notice them --- nevermind

LeifAndersen commented:
Okay @bennn I've added the id to the docs.

bennn commented on scribble-doc/scribblings/scribble/acmart.scrbl:
```diff
@@ -58,6 +58,7 @@ all mutually exclusive.}
+@defidform[noacm]
 @defidform[9pt]
 @defidform[10pt]
 @defidform[11pt]
```
There's also a `@tabular` of defaults below, copied from Table 2 (page 8) of the acmart pdf. `nonacm` should be in the tabluar.
https://www.acm.org/binaries/content/assets/publications/consolidated-tex-template/acmart.pdf

LeifAndersen commented:
Okay Ben. I have added it to the docs, and also added the other variables that acmart was missing. If we need any more variables I very much think we need a macro for the macro.

bennn commented on scribble-lib/scribble/acmart/lang.rkt:
```diff
@@ -116,7 +156,7 @@
            
            [body
-            #`(#%module-begin id (post-process #,review? #,screen? #,natbib? #,anonymous? #,authorversion? #,font-size #,format?) () . body)])))]))
+            #`(#%module-begin id (post-process #,review? #,screen? #,natbib? #,anonymous? #,authorversion? #,font-size #,nonacm? #,format?) () . body)])))]))
```
This line needs `#,timestamp? #,authordraft? #,acmthm?`
